HtmlEditor: add info about tabs and spaces (#5837)
DevExpress/devextreme-documentation: concepts/05 UI Components/HtmlEditor/05 Overview.md

        <!-- to -->
        <p>He<em>llo</em></p>

- HtmlEditor trims extra space and tab characters passed to the [value](/api-reference/10%20UI%20Components/dxHtmlEditor/1%20Configuration/value.md '/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/#value') option.

> - The component is not designed to convert text enclosed between curly brackets (`{}`) into a variable. Content between curly brackets is always treated as plain text.

Reviewer: ```suggestion
- The component is not designed to convert text enclosed in curly brackets (`{}`) to a variable. Content in curly brackets is always treated as plain text.
```
